src/main/kotlin/note/application/NoteSearcher.kt

```diff
@@ -13,6 +13,6 @@ class NoteSearcher(
      * @see Note
      * @return null or a note instance if it exists
      */
-    fun search(identifier: Identifier) = repository.search(identifier)
-
+    fun search(identifier: String) = repository.search(Identifier(identifier))
+    internal fun search(identifier: Identifier) = repository.search(identifier)
```

Comment:
The same happens here, you are testing the internal method instead of the public one. The fix is the same as in the above comment.

Reply:
This one should be internal, as we're using it from the delete use case.